docs/docs/etl-pipeline-tutorial/4-partition-asset.md

```diff
@@ -43,46 +42,27 @@ and deletes any previous value for that month. Copy the following asset under th
 Do not worry about the `automation_condition` in the `dg.asset` decorator for now. This is not necessary but will make more sense when we discuss automation later.
 :::
 
-## 2. Create a category-based partitioned asset
-
-Using known defined partitions is a simple way to break up your dataset when you know the different groups you want to subset it by. In our pipeline, we want to create an asset that represents the performance of each product category.
-
-1. To create the statically-defined partition for the product category, copy this code beneath the `monthly_sales_performance` asset:
-
-<CodeExample
-  path="docs_snippets/docs_snippets/guides/tutorials/etl_tutorial/src/etl_tutorial/defs/assets.py"
-  language="python"
-  startAfter="start_product_category_partition"
-  endBefore="end_product_category_partition"
-  title="src/etl_tutorial/defs/assets.py"
-/>
-
-2. Now that the partition has been defined, we can use that in an asset that calculates the product category performance:
-
-<CodeExample
-  path="docs_snippets/docs_snippets/guides/tutorials/etl_tutorial/src/etl_tutorial/defs/assets.py"
-  language="python"
-  startAfter="start_product_performance_asset"
-  endBefore="end_product_performance_asset"
-  title="src/etl_tutorial/defs/assets.py"
-/>
-
-## 3. Materialize partitioned assets
+## 2. Materialize partitioned assets
 
 To materialize these assets:
 
 1. Navigate to the assets page.
 2. Reload definitions.
 3. Select the `monthly_sales_performance` asset, then **Materialize selected**.
-4. Ensure all partitions are selected, then launch a backfill.
-5. Select the `product_performance` asset, then **Materialize selected**.
-6. Ensure all partitions are selected, then launch a backfill.
 
-TODO: Screenshot
+   When launching a partitioned asset, you will need to select the which partitions to execute. Because our partition has a start date (`"2018-01-01"`) and no end date, there will be partitions for every month from 2018. Only select partitions for the first four months of 2018. 
```

Comment:
```suggestion
   When materializing a partitioned asset, you will need to select which partitions to execute. Because our partition has a start date (`"2018-01-01"`) and no end date, there will be partitions for every month starting from 2018. Only select partitions for the first four months of 2018:
```